### ISSUE TITLE
[Wemos-D1] SDA and SCL are not connected to the I2C connectors

### ISSUE THREAD

**Issue by vongomben** [OfficineArduinoTorino/RuralHack#1]
As far as I can test, SDA and SCL on Wemos D1 (D1 B, D2 C) are not connected to the I2C Grove and Gravity pins. Is this possible or I'm missing something?

![image](https://user-images.githubusercontent.com/1215847/40856686-043ca186-65d9-11e8-9f2c-6c4a3dbced76.png)

![image](https://user-images.githubusercontent.com/1215847/40856706-13b2d7ca-65d9-11e8-8bde-b78d0009c394.png)

**Comment by kliment** [comment]
Yes, this is accurate, and it's my mistake. I'll put in a solder jumper to allow choosing between the two functions.

**Comment by kliment** [comment]
To modify the existing boards it's probably easiest to add wire jumpers to the non-D1 pads. Here are the necessary jumpers.

![selection_068](https://user-images.githubusercontent.com/228211/40982107-c07037cc-68dc-11e8-9988-0bc668b1e2e3.png)

**Comment by kliment** [comment]
Fixed in  ac0db75
